### PR TITLE
Add a section that provides instructions on migrating from deprecated methods to newer methods...

### PR DESCRIPTION
**Pull request by Zeemahh**
Currently, this only documents the two _main_ deprecated/out-of-date methods I've seen. Those being the `chatMessage` event for sending chat messages, and the `chatMessage` event for detecting commands.

### CODE REVIEW

File: content/scripting-manual/migrating-from-deprecated/creating-commands.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Creating commands
+weight: 432
+---
+
+## The `chatMessage` method
```
Comment (cerium-cfx) [inline content/scripting-manual/migrating-from-deprecated/creating-commands.md]: Perhaps indicate that this is now deprecated. E.g. "The `chatMessage` method (deprecated)"

File: content/scripting-manual/migrating-from-deprecated/creating-commands.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    for str in string.gmatch(inputstr, "([^" .. sep .. "]+)") do
+        t[i] = str
+        i = i + 1
+    end
+    return t
+end
+```
+
+## The [`RegisterCommand()`](https://runtime.fivem.net/doc/natives/#_0x5FA79B0F) -- which you should always be using
+This is a somewhat recent addition to the CFX natives, it is recommended to use this as it allow for the use of the integrated ace permissions system. This native consists of 3 parameters (`commandName`[string], `handler`[func] and `restricted`[boolean]).
```
Comment (cerium-cfx) [inline content/scripting-manual/migrating-from-deprecated/creating-commands.md]: Grammar mistake - `allow` -> `allows`

`ace` -> `ACE`

File: content/scripting-manual/migrating-from-deprecated/creating-commands.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+### Example
```
Comment (cerium-cfx) [inline content/scripting-manual/migrating-from-deprecated/creating-commands.md]: Perhaps mention somewhere that other examples can be found in the `Creating your first script` tutorials (Lua and C# respectively)

**Comment by Zeemahh** [comment]
Acted upon the requested changes and pushed the changes 😃

**Pull request merged**